src/main/java/core/basesyntax/figures/FigureSupplier.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.figures;
+
+import core.basesyntax.ColorSupplier;
+import java.util.Random;
+
+public class FigureSupplier {
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    int getRandomProperties() {
+        return new Random().nextInt(20);

```

Comment:
lets make Random as class level variable and lets make number 20 as constant field https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names